Src/Test.UnitTests.Sarif.PatternMatcher.Cli/AnalyzeCommandTests.cs

```diff
@@ -89,12 +92,66 @@ public void AnalyzeCommand_IntrafileBasic()
             region.StartColumn.Should().Be(2);
         }
 
+        [Fact]
+        public void AnalyzeCommand_FileSizeInKilobytes()
+        {
+            string rootDirectory = Directory.GetCurrentDirectory();
+            string smallTargetPath = Path.Combine(rootDirectory, SmallTargetName);
+            string largeTargetPath = Path.Combine(rootDirectory, LargeTargetName);
+
+            string definitionsText = GetIntrafileRuleDefinition();
+
+            string fileContents = "unused leading space  \r\n" +
+                                  " secret1              \r\n" +
+                                  " host1                \r\n" +
+                                  " id1                  \r\n" +
+                                  " secret2              \r\n" +
+                                  "unused trailing space \r\n";
+
+            SarifLog sarifLogWithLargeFileExcluded = RunAnalyzeCommandWithFileSizeInKilobytes(
+                definitionsText,
+                fileContents,
+                fileSizeInKilobytes: 1);
+
+            SarifLog sarifLogWithLargeFileIncluded = RunAnalyzeCommandWithFileSizeInKilobytes(
+                definitionsText,
+                fileContents,
+                fileSizeInKilobytes: 1024);
```

Comment:
@marmegh Perhaps a naive question, but what happens if I pass a value like 999999999999?

Reply:
See the corresponding PR for the SARIF sdk [#2494](https://github.com/microsoft/sarif-sdk/pull/2494) for where this is now defined.

The actual input is text that the SARIF sdk attempts to convert to an int. When that fails, SPAM returns the following errors message:
`Option 'file-size-in-kb' is defined with a bad format.`
Followed by help content describing intended usage of each option.